log/logger.go

```
@@ -28,10 +33,19 @@ type logEntry struct {
 	Values []interface{}
 }
 
-// NewLogger returns a new instance of the clusterctl.
+// NewLogger returns a new instance of the logger.
 func NewLogger() LoggerImpl {
+	logThresold := defaultLogThreshold
+	requestedLogThresoldStr := os.Getenv(EnvTanzuCLILogLevel)
+	if requestedLogThresoldStr != "" {
+		requestedLogThresold, err := strconv.ParseUint(requestedLogThresoldStr, 10, 32)
+		if err == nil {
```

What if user set incorrect value, and we are not capturing the err here, ignoring the error?
IMO, at minimal we need to warn the error message so that user knows that incorrect value has been set.